inc/wp-forms-api.js

```diff
@@ -13,13 +13,49 @@
 		},
 	});
 
+	function loadMCETemplates(content, editor) {
+		wp.ajax.post('wp_form_get_mce_templates', {
+			content: content,
+			type: getUserSetting('editor')
```

Comment:
Yeah this isn't complete yet, I was just getting the base code in there – probably not ready for a full review yet~!